docker/qgisserver/geomapfish_qgisserver/accesscontrol.py

```diff
@@ -36,7 +38,17 @@ def create_session_factory(url, configuration):
     engine = sqlalchemy.create_engine(url, **configuration)
```

Comment:
As I understand, create_engine(autocommit=True) should avoid sending BEGIN, so we won't get idle in transaction.